src/Equinor.ProCoSys.IPO.Command/Validators/InvitationValidators/InvitationValidator.cs

```diff
@@ -300,8 +303,33 @@ public async Task<bool> ValidSigningParticipantExistsAsync(int invitationId, int
             return participant.AzureOid == _currentUserProvider.GetCurrentUserOid();
         }
 
-        public async Task<bool> SameUserUnCompletingThatCompletedAsync(int invitationId, CancellationToken cancellationToken)
+        public async Task<bool> ValidUnsigningParticipantExistsAsync(int invitationId, int participantId, CancellationToken cancellationToken)
         {
+            var participant = await (from p in _context.QuerySet<Participant>()
+                                     where EF.Property<int>(p, "InvitationId") == invitationId &&
+                                           p.Id == participantId &&
+                                           p.SortKey > 1 &&
+                                           (p.Organization == Organization.TechnicalIntegrity ||
```

Comment:
Personally I usually do the checks againt the values that give the fewest lines. So instead (p.Organization != Organization.External && p.Organization != Organization.Supplier). Don't need to change it here, this is ok.